apache-tomcat8/hooks/init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+function debug { if [ ! -z "$DEBUG" ]; then echo "$*"; fi }
+
+echo "Preparing TOMCAT_HOME..."
+
+debug "\$sys.ip        = {{sys.ip}}"
```

Comment:
The leading `\$`s probably aren't needed here, since these are not Bash variables anyway.